src/tools/singlejar/combiners.cc

```diff
@@ -15,6 +15,8 @@
 #include "src/tools/singlejar/combiners.h"
 #include "src/tools/singlejar/diag.h"
 
+#include <cctype>
```

Comment:
_Internal reviewer comment_: Is this change needed?

Reply:
Yes. Needed by `std::isspace` in this file. glibc probably includes `<cctype>` transitively from other standard header files, but not for MSVC.

Reply:
_forwarded_